tests/src/test/java/tachyon/shell/command/ChgrpCommandTest.java

```diff
@@ -41,4 +41,20 @@ public void chgrpTest() throws IOException, TachyonException {
     group = mFileSystem.getStatus(new TachyonURI("/testFile")).getGroupName();
     Assert.assertEquals("group2", group);
   }
+
+  @Test
+  public void chgrpRecursiveTest() throws IOException, TachyonException {
+    clearLoginUser();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testDir/testFile", WriteType.MUST_CACHE, 10);
+    // "chgrp -R group1 /testDir" should apply to both dir and child file
+    mFsShell.run("chgrp", "-R", "group1", "/testDir");
+    String group = mFileSystem.getStatus(new TachyonURI("/testDir")).getGroupName();
+    Assert.assertEquals("group1", group);
+    group = mFileSystem.getStatus(new TachyonURI("/testDir/testFile")).getGroupName();
+    Assert.assertEquals("group1", group);
+    // chgrp to another group.
+    mFsShell.run("chgrp", "-R", "group2", "/testDir");
```

Comment:
No, I think there is a problem in DefaultParser with stopAtNonOption that -options must come before arguments.
I think it's fine here because the usage clearly says "chgrp -R <group> <path>"